include/Battlemines_2/board.h

```diff
@@ -5,6 +5,15 @@
 #include <unordered_set>
 #include <vector>
 
+const int minWidth = 5;
```

Comment:
#should use a naming convention that distinguishes these compile time constants from normal variables

#should declare them as `static constexpr`  instead